internal/services/patch_anchor_platform_transactions_completion.go

```diff
@@ -102,18 +193,10 @@ func (s *PatchAnchorPlatformTransactionCompletionService) PatchTransactionComple
 			return err
 		}
 	} else {
-		err = fmt.Errorf("PatchAnchorPlatformTransactionService: invalid payment status to patch to anchor platform. Payment ID: %s - Status: %s", payment.ID, paymentStatus)
+		err := fmt.Errorf("PatchAnchorPlatformTransactionService: invalid payment status to patch to anchor platform. Payment ID: %s - Status: %s", payment.ID, payment.Status)
```

Comment:
👋 In Go, the `:` char is used to wrap errors and it's relied on when you want to debug the stack trace of an error stack. At a first glance, one would consider anything separated by `:` as different errors in an error stack, so when `:` is added many times like this, it causes confusion and can make the debugging harder.